Content.Server/Nutrition/Components/SliceableFoodComponent.cs

```diff
@@ -70,7 +70,7 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
                 }
             }
 
-            SoundSystem.Play(Filter.Pvs(Owner), _sound.GetSound(), Owner.Transform.Coordinates,
```

Comment:
Play this sound on `eventArgs.User` instead as this entity might be deleted after this.